roles/infra/defaults/main.yml

```diff
@@ -19,3 +19,7 @@ cluster_gateway_user: "{{ cluster_ssh_user }}"
 
 # Adopt infrastructure into the in-memory inventory
 terraform_adopt_inventory: true
+
+# Disable checks for force_path_style s3 option
+disable_s3_force_path_style_checks: false
```

Comment:
Can we give this a `terraform_` prefix? Else if it pops up in a config no one will ever find what it applies to.

Reply:
Good point - updated.